Gems/PhysX/Code/Source/HeightfieldColliderComponent.h

```diff
@@ -70,6 +77,10 @@ namespace PhysX
         AZStd::string GetCollisionGroupName() override;
         void ToggleCollisionLayer(const AZStd::string& layerName, AZ::Crc32 filterTag, bool enabled) override;
 
+        // AZ::Data::AssetBus::Handler overrides ...
+        void OnAssetReady(AZ::Data::Asset<AZ::Data::AssetData> asset);
+        void OnAssetError(AZ::Data::Asset<AZ::Data::AssetData> asset);
```

Comment:
Does this need an OnAssetReload() as well for the case that you overwrite an existing baked asset with a new one?